api/v1/countries.py

```diff
@@ -34,10 +35,18 @@ async def get_names(request: Request, country_state: CountryStateDep, aed_state:
         for _ in range(len(countries)):
             country, count = await receive_stream.receive()
             country_count_map[country.name] = count
+
+    def limit_country_names(names: dict[str, str]):
+        if target_language is None:
+            return names
+        if target_language in names:
+            return {target_language: names[target_language]}
+        return names
```

Comment:
This could be simplified and optimized (avoid double dict lookup) :slightly_smiling_face: 
```suggestion
        if language and (name := names.get(language)):
            return {language: name}
        return names
```